scripts/docker-cross-env.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# This script is used in the Dockerfile to run the build commands with the correct environment variables
+# This maps the docker "$TARGETPLATFORM" to relevant npm and cargo environment variables
+
+set -eux
+
+if [ "$#" -lt 2 ]; then
+  echo "usage: $0 [platform] [cmd...]" >&2
+  exit 1
+fi
+
+PLATFORM="$1"
+shift
+
+if [ "$PLATFORM" = "linux/arm64" ] || [ "$PLATFORM" = "linux/arm64/v8" ]; then
+  export npm_config_target_platform=linux
+  export npm_config_target_arch=arm64
+  export npm_config_target_libc=glibc
```

Comment:
Are these variables the ones you mentioned in https://github.com/matrix-org/matrix-rust-sdk-crypto-nodejs/pull/8 as being understood by `node-gyp`?